bin/upgrading/base-data.sql

```diff
@@ -1,5 +1,4 @@
-# This file is automatically generated from MySQL by $LJHOME/bin/dumpsql.pl
-# Don't submit a diff against a hand-modified file - dump and diff instead.
+# This file is no longer automatically generated by $LJHOME/bin/dumpsql.pl
```

Comment:
interesting, I guess I'm not opposed, but I'm curious on this. did the maintenance burden of managing dumpsql to add the profile services table feel too high?

Reply:
We found that dumpsql.pl would no longer have any effect since you removed schematables 9 years ago.

Reply:
love it. good bye, dumpsql, good bye.